x-pack/plugins/security_solution/public/common/components/events_viewer/index.tsx

```diff
@@ -569,17 +566,15 @@ const StatefulEventsViewerComponent: React.FC<EventsViewerProps & PropsFromRedux
                             unitCountText={unitCountText}
                             browserFields={browserFields}
                             data={nonDeletedEvents}
-                            disabledCellActions={FIELDS_WITHOUT_CELL_ACTIONS}
+                            disableCellActions={disableCellActions}
```

Comment:
I think we should also remove file `x-pack/plugins/security_solution/public/common/lib/cell_actions/constants.ts` because its `FIELDS_WITHOUT_CELL_ACTIONS` is not in use any more